x-pack/legacy/plugins/siem/public/mock/global_state.ts

```diff
@@ -69,6 +69,16 @@ export const mockGlobalState: State = {
   network: {
     page: {
       queries: {
+        [networkModel.NetworkTableType.topCountriesDestination]: {
+          activePage: 0,
+          limit: 10,
+          topCountriesSort: { field: NetworkTopNFlowFields.bytes_out, direction: Direction.desc },
```

Comment:
I think I mentioned this in the gql type definition down below where it's defined, but this is one of those places where you have to do a double-take to make sure this is the right type.